crates/ruff_linter/src/rules/pyupgrade/rules/yield_in_for_loop.rs

```diff
@@ -100,13 +100,14 @@ pub(crate) fn yield_in_for_loop(checker: &mut Checker, stmt_for: &ast::StmtFor)
         checker
             .semantic()
             .current_scope()
-            .get_all(name.id.as_str())
```

Comment:
This was retrieving all shadowed bindings, which seems way overkill and leads to false negatives.